pom.xml

```diff
@@ -317,7 +317,7 @@
         <dependency>
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-ingest-sdk</artifactId>
-            <version>1.0.2-beta.7</version>
```

Comment:
can we add this version change in the title or something? just thinking in terms of rel notes, we'd wanna call out this upgrade. but afaik rel notes only pulls the titles of each commit?

Reply:
Yes, can add it in title. Thanks for call out. [Have added in description though]